### DISCUSSION TITLE
Battery and Propeller test fails in Cf-client

### DISCUSSION THREAD

**Discussion by WiresharkIO** [bitcraze/discussions#535] [Q&A] [answered]
Hello

I was wondering what could be the reason of battery and propeller test fails in cf-client. 

console:
HEALTH: Propeller test on M2 [FAIL]. low: 0.0, high: 2.50, measured: 4.56
HEALTH: Battery sag during load test (1.24 > 0.95) [FAILED].

I changed the M2 motor and the battery but the test still fails. 

Any inference or suggestion.

**Comment by tobbeanton** [comment, marked as answer]
When the propeller test fail it is most likely an unbalances/damaged propeller but could also be a bent motor shaft. The threshold as only a guidance and it might fly just as well when above it.

When the battery test fails it is usually oxidation in the connector or a bad/old battery. You can read about it in [this](https://www.bitcraze.io/2021/02/experimental-battery-testing-feature/) blog post. But the same here, the threshold as only a guidance.

**Reply by WiresharkIO** [reply]
@tobbeanton Thank you
